fe/package.json

```diff
@@ -6,14 +6,13 @@
     "@apollo/client": "^4.0.0",
     "@graphql-tools/schema": "^10.0.0",
     "@types/autosuggest-highlight": "^3.2.3",
-    "@types/uuid": "^10.0.0",
```

Comment:
`uuid` ships it's own types and this is deprecated. 
https://www.npmjs.com/package/@types/uuid?activeTab=code